packages/roosterjs-content-model/lib/modelApi/selection/deleteSelections.ts

```diff
@@ -111,6 +118,8 @@ const deleteSelectionStep1: DeleteSelectionStep = (context, options, model) => {
             let markerFormat = model.format;
             let insertMarkerIndex = 0;
 
+            saveDeletedElements(block, context, tableContext, deleteResult);
```

Comment:
It is a good idea to return deleted content when delete. But I think this process can be integrated into the deleting process, but not let it be standalone here. You can move the deleted content into the target model when delete it, like L151 and L160